pkg/v1/tkg/web/src/app/views/landing/wizard/shared/components/steps/identity-step/identity-step.component.html

```diff
@@ -594,11 +594,11 @@ <h3 class="modal-title">Verify LDAP Configuration</h3>
                                             <clr-icon aria-live="assertive" a11yTooltipTrigger shape="info-circle" size="22">
                                             </clr-icon>
                                             <clr-tooltip-content clrPosition="top-right" clrSize="lg" *clrIfOpen>
-                                                <span>A user name used to test LDAP user search functionality.</span>
+                                                <span>Enter the CN attribute for the user to test LDAP user search functionality.</span>
                                             </clr-tooltip-content>
                                         </clr-tooltip>
                                     </label>
-                                    <input clrInput formControlName="testUserName" placeholder="userid" name="testUserName" />
+                                    <input clrInput formControlName="testUserName" placeholder="CN" name="testUserName" />
```

Comment:
Could we make the placeholder `CN attribute` something like `Test User` or `User's CN Attribute`?